src/js/pages/services/DeploymentsTab.js

```diff
@@ -53,12 +38,123 @@ class DeploymentsTab extends mixin(StoreMixin) {
     this.store_listeners = [
       {name: 'dcos', events: ['change'], suppressUpdate: true}
     ];
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    }, this);
   }
 
   onDcosStoreChange() {
     this.setState({deployments: DCOSStore.deploymentsList});
   }
 
+  renderAffectedServices(prop, deployment) {
+    return (
+      <dl className="deployment-services-list flush-top flush-bottom tree-list">
+        <dt className="deployment-id text-uppercase">{deployment.id}</dt>
+        {this.renderAffectedServicesList(deployment.getAffectedServices())}
```

Comment:
in case services is empty (for whatever reason) and we get back 0 `dd` elements, does the layout break? (I remember we got bit by this in some cases – but maybe it's not an issue in Canvas?)